### PR TITLE
Feat: Added top navigation bar

### PR DESCRIPTION
**Pull request by LordUttam**
Added a standard navigation bar which contains a search bar and links. 
image

### CODE REVIEW

File: pages/docs.html [hunk 1]
```diff
@@ -1322,6 +1322,52 @@ <h1 class="main__heading p--1" id="modal">
                     </div>
                 </div>
             </article>
+
+
+            <h1 class="main__heading p--1" id="topnav">
```
Comment (mcmonojit) [inline pages/docs.html:1327]: Underscore character is not used in class naming convention. You can use '-' instead. Do look into this.

**Review by mcmonojit — APPROVED** [review]
Do look into the class naming convention and accordingly modify the class names throughout the file. Rest of the code looks good.

File: pages/docs.html (hunk 1 above)
Comment (LordUttam) [reply pages/docs.html:1327]: BEM (block__element--modifier) is a standard class naming convention and I have used it throughout CL.

File: pages/docs.html [hunk 2]
```diff
@@ -1322,6 +1322,52 @@ <h1 class="main__heading p--1" id="modal">
...
+                            style="width: 700px; height: 983px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (Mohasin-Haque) [inline pages/docs.html:1365]: You should not use inline styling

File: styles/navigation.css [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  color: var(--base-color);
+}
+
+.topnav__btn {
+  position: relative;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
```
Comment (Mohasin-Haque) [inline styles/navigation.css]: you can remove whitespaces if you forgot

**Review by Mohasin-Haque — COMMENTED** [review]
Your code is long due to one file. Its good to see that how you managed it. You can also work on the suggestions and merge.

File: pages/docs.html (hunk 2 above)
Comment (LordUttam) [reply pages/docs.html:1365]: It is from carbon's iframe. I did not make it. It is automatically there if I copy from carbon.now.sh.

File: styles/navigation.css (hunk 3 above)
Comment (LordUttam) [reply styles/navigation.css]: When I have a lot of styles in a component I prefer to separate display, position and other properties using a whitespace line between them. But I can change it if you say so.

**Pull request merged**